python_modules/dagster/dagster/core/execution/plan/external_step.py

```diff
@@ -180,11 +220,14 @@ def external_instance_from_step_run_ref(
         DagsterInstance: A DagsterInstance that can be used to execute an external step.
     """
     instance = DagsterInstance.ephemeral()
-    # re-execution expects the parent run to be available on the instance, so add these
-    if step_run_ref.parent_run:
+    # re-execution expects the parent run(s) to be available on the instance, so add these
+    for run in step_run_ref.run_group or []:
```

Comment:
nit: i think with `check.opt_list_param(...)`, run_group should've already been an empty list if no group